src/types.js

```diff
@@ -99,6 +100,11 @@ export type IOSNativeProps = $ReadOnly<{|
    * instance, to show times in Pacific Standard Time, pass -7 * 60.
    */
   timeZoneOffsetInMinutes?: ?number,
+
+  /**
+   * The date picker text color.
+   */
+  textColor?: ColorValue,
```

Comment:
```suggestion
  textColor?: ?ColorValue,
```

let's please have this optional